app/models/models.js

```diff
@@ -2270,7 +2270,7 @@ YUI.add('juju-models', function(Y) {
           serviceData.num_units = 0;
         } else {
           // Test models or ghosts might not have a units LazyModelList.
-          serviceData.num_units = units && units.size() || 1;
+          serviceData.num_units = units && units.size() || 0;
```

Comment:
on line 2270 above is where it handles the number of units for subordinates. I believe this was also QA'd to work.